### PR TITLE
Tile Dashboard service methods

### PR DESCRIPTION
**Pull request by blackbaud-conorwright**
Issue: blackbaud/skyux2#2115

Also https://github.com/blackbaud/skyux2/issues/2284 and https://github.com/blackbaud/skyux2/issues/2283.

### CODE REVIEW

File: src/app/components/tile/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+      The <stache-code>sky-tile</stache-code> component creates a collapsible container that serves as a building block for pages and forms in SKY UX applications. Within the <stache-code>sky-tile</stache-code> component, the <stache-code>sky-tile-title</stache-code> component displays a header for the tile, the <stache-code>sky-tile-summary</stache-code> component displays summary information, and the <stache-code>sky-tile-content</stache-code> component displays the tile content. The <stache-code>sky-tile-content-section</stache-code> component creates a padded section inside a <stache-code>sky-tile-content</stache-code> element. The <stache-code>sky-tile-dashboard</stache-code> component creates a container to group multiple tiles. To display a <stache-code>sky-tile</stache-code> component within a <stache-code>sky-tile-dashboard</stache-code> component, you must <a routerLink="/learn/get-started/advanced/add-entry-components">include it as an entry component</a> in the application module, and the selector for the tile component must begin with <stache-code>div.</stache-code> followed by a custom selector.
```
Comment (Blackbaud-BobbyEarl) [inline src/app/components/tile/index.html]: Not trying to introduce scope creep, but any objections to throwing `and the selector for the tile component must begin with div. followed by a custom selector.` into a `sky-alert` or making it bold / stand out more?  I've seen a few engineers get bit by this nuance in the past.

Comment (blackbaud-johnly) [reply src/app/components/tile/index.html]: Thanks for the suggestion. I will do this now.
